### DISCUSSION TITLE
Runtime 的翻譯

### DISCUSSION THREAD

**Discussion by mattwang44** [python/python-docs-zh-tw#507] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

Runtime

### 術語翻譯

不翻譯或「執行環境」

### 說明

- 出處：是個普遍詞彙，許多文件都有出現
- 翻譯：
  - 未有統一且常用的中文翻譯。 
  - 我目前都將它翻為「執行環境」，想不到更好的翻譯，但也覺得不夠貼切。

### 參考資料

- wikibooks 對照表翻譯為「執行環境」([ref](https://zh.wikibooks.org/zh/%E5%A4%A7%E9%99%86%E5%8F%B0%E6%B9%BE%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7%E8%A1%A8))
- microsoft 文件翻譯為「執行期間」、「執行階段」，並不統一
- 教育部樂詞網為「運轉時間」、「執行期」、「運行時間」、「執行時間」，也沒有統一

**Comment by cschan1828** [comment]
個人傾向不翻譯，直接用 runtime，避免更多誤解。
